packages/block-library/src/query-title/edit.js

```diff
@@ -89,16 +93,35 @@ export default function QueryTitleEdit( {
 		titleElement = (
 			<>
 				<InspectorControls>
-					<PanelBody title={ __( 'Settings' ) }>
-						<ToggleControl
-							__nextHasNoMarginBottom
+					<ToolsPanel
+						label={ __( 'Settings' ) }
+						resetAll={ () =>
+							setAttributes( {
+								showPrefix: false,
+							} )
+						}
+					>
+						<ToolsPanelItem
+							hasValue={ () => showPrefix !== undefined }
 							label={ __( 'Show archive type in title' ) }
-							onChange={ () =>
-								setAttributes( { showPrefix: ! showPrefix } )
+							onDeselect={ () =>
+								setAttributes( { showPrefix: false } )
 							}
-							checked={ showPrefix }
-						/>
-					</PanelBody>
+							resetAllFilter="core/filter-reset-all-settings"
+							isShownByDefault
```

Comment:
```suggestion
							isShownByDefault
```

I don't think resetAllFilter is necessary.